### ISSUE TITLE
PLUMBER 2 Refactoring - testing

### ISSUE THREAD

**Issue by TeaganKing** [ESCOMP/CTSM#2411]
- [ ] Add a unit test for subset_data for the latest addition
- [ ] Add a system test for a PLUMBER2 site US-Ha?
These are from #2155

**Comment by ekluzek** [comment]
@TeaganKing in ctsm5.3.0 we have PLUMBER2 surface datasets that can be used. But, we aren't going to test any of them since these tests aren't already in. 

You could try testing with the new files if you like, or we'll add the testing above on b4b-dev in a few weeks. We'll respond to problems at that time. Since other things are working it seems likely that they are all fine.

**Comment by wwieder** [comment]
I don't think we should ask Teagan to do more testing at this point.  Let's work on this after the 5.3 dust settles.

**Comment by samsrabin** [comment]
We should also test at least some of the sites that had to be fixed with PR #4092.

**Comment by ekluzek** [comment]
It turns out this is mostly done now. With the exception of some run_tower testing for PLUMBER2
